eunkyung/controllers/userController.js

```diff
@@ -0,0 +1,24 @@
+const userService = require("../services/userService");
+
+const createUser = async (req, res) => {
+  const { name, password, email, profileImage } = req.body;
+
+  await userService.createUser(name, password, email, profileImage);
+  return res.status(201).json({ message: "userCreated" });
+};
+
+const login = async (req, res) => {
+  const { email, password } = req.body;
+  try {
+    const loginSuccess = await userService.login(email, password);
+
+    return res.status(200).json({ accessToken: loginSuccess });
+  } catch (err) {
+    return res.status(400).json({ message: err.message });
+  }
```

Comment:
- 해당 부분도 마찬가지로 email, password가 없다면 `key error`를 반환하는 예외처리가 필요할 것 같습니다!
- loginSuccess라고 되어 있는 부분의 변수명을 수정해주면 좋을 것 같아요
```suggestion
  try {
    const { email, password } = req.body;
    
    if( !email || !passrod) {
      throw new Error("Key Error!");
    }
    
    const token = await userService.login(email, password);

    return res.status(200).json({ accessToken: token });
  } catch (err) {
    return res.status(400).json({ message: err.message });
  }
```